Chinese/Yifan/modernCity.md

```diff
@@ -2,7 +2,4 @@
 至尊宝：我勒个去》》》》这是神马情况。。。
 （他走到一个卖祖传贴膜的哥们前。。。）
 
-
-请问大哥：
-
-
+请问大哥：[盘丝洞怎么走?](../sell_film/sell_film.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.